test/models/post_test.rb

```diff
@@ -21,4 +21,16 @@ class PostTest < ActiveSupport::TestCase
     "own idea.", post.og_description
     assert_equal "http://www.thefirehoseproject.com/assets/meta-tag-coder.png", post.og_image
   end
+
```

Comment:
I personally don't think validations are worth testing, but if you want to you might want to look into `shoulda-matchers` gem.

https://github.com/thoughtbot/shoulda-matchers

Allows you to write tests like:

```
  should validate_presence_of(:name)
```

I suggest either deleting these tests, or adding the shoulda-matchers gem to write code in a clearer way.